wordpress-mobile/WordPress-Aztec-Android
# Issue/181 update placeholder images

### Fix
Add placeholder image attributes to `AztecText` to enable custom placeholder images in client apps.

### Test
1. Run demonstration app.
2. Notice [`ic_image_failed`](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/res/drawable/ic_image_failed.xml) and [`ic_image_loading`](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/res/drawable/ic_image_loading.xml) placeholder images.
3. Add `aztec:drawableFailed` and `aztec:drawableLoading` to [`AztecText`](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/app/src/main/res/layout/activity_main.xml#L26).
4. Run demonstration app.
5. Notice new placeholder images.
